### ISSUE TITLE
Crashing on Lenovo Legion 5 Pro 16ARH7H

### ISSUE THREAD

**Issue by absh959595** [4JX/L5P-Keyboard-RGB#123]
On `Lenovo Legion 5 Pro 16ARH7H`, build [v0.19.0](https://github.com/4JX/L5P-Keyboard-RGB/pull/117) crashes
```bash
>>>sudo ./legion-kb-rgb --gui
No subcommands found, starting in GUI mode. To view the possible subcommands type "legion-kb-rgb --help".
The application panicked (crashed).
Message:  called `Result::unwrap()` on an `Err` value: D-Bus error: /usr/bin/dbus-launch terminated abnormally without any error message (org.freedesktop.DBus.Error.Spawn.ExecFailed)
Location: /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ksni-0.1.3/src/service.rs:83

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```
relevant lsusb are:
```bash
>>>lsusb | grep 048d
Bus 005 Device 002: ID 048d:c975 Integrated Technology Express, Inc. ITE Device(8295)
Bus 001 Device 002: ID 048d:c102 Integrated Technology Express, Inc. ITE Device(8910)
```

**Comment by 4JX** [comment]
> ksni-0.1.3

I recon this is part of the dependencies I use to make the icon tray shenanigans, can you verify if just using CLI arguments without the GUI works?

**Comment by absh959595** [comment]
Tested `sudo ./legion-kb-rgb set -e Static -c 255,0,0,255,0,0,255,0,0,255,0,0` and that does work. Maybe warn in README? 
This probably also resolves (https://github.com/4JX/L5P-Keyboard-RGB/issues/119)
Thanks.

**Comment by 4JX** [comment]
`16ARH7H` is a 2022 model that has been supported for a while now, that issue is for a 2023 model where the corresponding PR to fix it is #120 but I'm waiting for another person to test it before merging

I probably have to find some other create-a-tray-icon library or just depend on the non-forked version which isn't ideal since that depends on gtk.

**Comment by 4JX** [comment]
On a better look this seems like it is a system specific error.

You should try solutions for [`/usr/bin/dbus-launch terminated abnormally without any error message`](https://duckduckgo.com/?q=%2Fusr%2Fbin%2Fdbus-launch+terminated+abnormally+without+any+error+message&va=n&ia=web) and hopefully something will work.
